modules/code-builder/src/brick/design0/BrickStatement.ts

```diff
@@ -52,6 +52,51 @@ export default class BrickStatement extends BrickModelStatement {
     }
 
     public get SVG(): string {
-        return '';
+        const argsLength = Object.keys(this._args).length;
+        const repeatPath = 'v 17 h -4 v -3 h -4 v 10 h 4 v -3 h 4 '.repeat(argsLength - 1);
+
+        return `<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+                <svg
+                width="85.5"
+                height="66"
+                version="1.1"
+                id="svg474"
```

Comment:
We can replace these hardcoded ids with a combination of `tag-type` and `uuid`, should I do that?

Reply:
ignore these for now ... design aspects are priority